gradle.properties

```diff
@@ -1,9 +1,9 @@
-projectVersion=2.1.1.BUILD-SNAPSHOT
+projectVersion=2.2.0.BUILD-SNAPSHOT
```

Comment:
Please do not change this because if for some reason we need to revert the commit, we don't want to revert the project version. Once this is merged I'll update the project version in a separate commit (and also the core branch). 
Then a new 2.2.0 version of this module will be included in Micronaut 2.2.0

Reply:
OK. I'll change it back and push the change.